packages/flutter_tools/test/general.shard/test/test_compiler_test.dart

```diff
@@ -152,12 +152,12 @@ environment:
     final File generatedMain = fileSystem
       .directory('.dart_tool')
       .childDirectory('flutter_build')
-      .childFile('generated_main.dart');
+      .childFile('dart_plugin_registrant.dart');
 
     expect(generatedMain, exists);
     expect(
-      generatedMain.readAsLinesSync(),
-      contains("import 'test/foo.dart' as entrypoint;")
+      generatedMain.readAsStringSync(),
+      contains('APlugin.registerWith();')
```

Comment:
this could be the entire content

Reply:
I tried to keep the scope similar to what was there before.  Asserting the whole content while being more exact is more fragile, that's why I chose just to assert the `registerWith` call.